.ci/load/Jenkinsfile

```
@@ -0,0 +1,179 @@
+// For documentation on this pipeline, please see the README.md in this directory
+pipeline {
+    agent any
+    environment {
+        REPO = 'apm-agent-java'
+        APP = 'spring-petclinic'
+        APP_BASE_DIR = "src/${APP}"
+        METRICS_BASE_DIR="metrics/"
+        AGENT_BASE_DIR = "agent/"
+        ORCH_URL = 'obs-load-orch.app.elastic.co:8000'
+        // Set below for local development
+        // ORCH_URL='10.0.2.2:8000'
+        DEBUG_MODE = '0' // set to '0' for production
+        // This is a placeholder server. This will change to a dummy APM Server when it is available.
+        APM_SERVER_URL = 'https://2a2bd0e2806a47e5996eeeec6d22e6df.apm.eu-west-3.aws.elastic-cloud.com:443'
+
```

If this is a MBP, then I'd suggest to also move the params to env variables to bypass an existing issue with the very first builds.


```suggestion
        LOCUST_RUN_TIME = "$(params.duration}"
        LOCUST_USERS = "$(params.concurrent_requests}"
```

---

This won't be a MBP so I think we're OK here.